os-fpga/yosys_verific_rs
# EDA2762 - Support auto-assign internal signal

First iteration to support auto-assign internal signal

Testing done:
- Port this PR to latest NS Raptor
- Run test/batch, test/batch_gen2 and test/batch_gen3
- Run all GJC - make sure compilation pass but _**I think**_ there is issue of the project/overall-architecture
